File: smart_travel_FE/lib/domain/entities/banner.dart
class BannerEntity {
  final int id;
  final String title;
  final String imageUrl;
  final String? linkUrl;
  final String? description;
  final bool active;

  BannerEntity({
    required this.id,
    required this.title,
    required this.imageUrl,
    this.linkUrl,
    this.description,
    required this.active,
  });
}

File: smart_travel_BE/src/main/java/com/example/smart_travel_BE/entity/Banner.java
package com.example.smart_travel_BE.entity;
import jakarta.persistence.*;
import lombok.*;
import lombok.experimental.FieldDefaults;

@Entity
@Table(name = "banners")
@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
@FieldDefaults(level = AccessLevel.PRIVATE)
public class Banner {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    Long id;

    String title;       // Tiêu đề banner
    String imageUrl;    // Link ảnh banner
    String linkUrl;     // Link điều hướng khi click vào banner
    String description; // Mô tả ngắn về chương trình du lịch
    boolean active;     // Trạng thái hiển thị
}